src/views/v2/month/calendar-body/day/calendar-events/calendar-event/tooltip/title.php

```diff
@@ -20,10 +20,13 @@
 <h3 class="tribe-events-calendar-month__calendar-event-tooltip-title tribe-common-h7">
 	<a
 		href="<?php echo esc_url( $event->permalink ) ?>"
-		title="<?php echo esc_attr( get_the_title( $event->ID ) ); ?>"
+		title="<?php echo esc_attr( $event->title ); ?>"
 		rel="bookmark"
 		class="tribe-events-calendar-month__calendar-event-tooltip-title-link tribe-common-anchor-thin"
 	>
-		<?php echo wp_kses_post( get_the_title( $event->ID ) ); ?>
+		<?php
+		// phpcs:ignore
+		echo $event->title;
```

Comment:
https://github.com/moderntribe/the-events-calendar/pull/2987/files?utf8=%E2%9C%93&diff=split&w=1#r366022101